app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/AlertPlayer.java

```diff
@@ -115,6 +115,7 @@ boolean isUiThread() {
     // when ascending how many minutes since alert started do we wait before escalating
     final static int MAX_VIBRATING_MINUTES = 2;
     final static int MAX_ASCENDING_MINUTES = 5;
+    public int Vibrating_Calc = Pref.getBoolean("delay_ascending_3min", true) ? MAX_VIBRATING_MINUTES : -1; // Set to 2 if enabled and -1 if disabled
```

Comment:
This comment is also wrong because the variable is set to a predefined constant and not to 2.

Reply:
Corrected it.